scalalib/src/mill/javalib/android/AndroidSdkModule.scala

```diff
@@ -118,6 +120,25 @@ trait AndroidSdkModule extends Module {
     PathRef(buildToolsPath().path / "apksigner")
   }
 
+  /**
+   * Provides the path for the Android Debug Bridge (adt) tool.
+   *
+   * For more information, refer to the official Android documentation [[https://developer.android.com/tools/adb]]
+   */
+
+  def adbPath: T[PathRef] = Task {
+    PathRef(sdkPath().path / "platform-tools" / "adb")
+  }
+
+  /**
+   * List all attached devices and emulators.
+   *
+   * For more information, refer to the official Android documentation [[https://developer.android.com/tools/adb]]
+   */
+  def listAvailableDeviceTargets: Target[CommandResult] = Task {
+    os.proc(adbPath().path / "-l").call()
```

Comment:
This should probably be `os.proc((adbPath().path, "-l")).call()`, since `-l` is a CLI flag and not a filesystem path segment

Reply:
ah this was a leftover sorry, instead I deleted it and added the necessary methods that are needed to run android tests